### PR TITLE
Fixing collision with tmp dir.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Comment by Horneth** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/165/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_jes.wdl [hunk 1]
```diff
@@ -70,8 +70,7 @@ task modify_file_gcs {
     Boolean ready
     String file_path_raw
     command {
-        echo "override" > tmp
-        gsutil cp tmp ${file_path_raw}
+        echo "override" | gsutil cp - ${file_path_raw}
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_jes.wdl:73]: Actually I think I'd prefer this stay as it is and the tmpdir naming should be made collision-resistant over in Cromwell.

Comment (kshakir) [reply src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_jes.wdl:73]: Perhaps when that bug is fixed, you can submit another PR to put it back later? 😉

**Comment by mcovarr** [comment]
:+1: yeah I guess  😛 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/165/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_jes.wdl (hunk 1 above)
Comment (geoffjentry) [reply src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_jes.wdl:73]: Yeah priority right now is unblocking Chris' pr

**Pull request merged**